src/lib/users/clients/auto.rb

```diff
@@ -116,6 +118,25 @@ def check_users(users)
 
         Yast::Report.Error(_("Found users in profile with equal <uid>.")) if report
       end
+
+      def read_linuxrc_root_pwd(config)
+        root_user = config.users.root
+        # use param only if profile do not already contain it
+        return if root_user&.password
+
+        # root user not defined
+        if !root_user
```

Comment:
This is not the first time we need to populate a config with a root user, see https://github.com/yast/yast-users/blob/y2users/src/lib/y2users/users_simple/reader.rb#L71.

Maybe we need a method to generate the default root user. Otherwise, we can end up with root users populated with different data. What do you think?

Reply:
in fact we do not need default root user but just set root user password. so maybe helper in UsersCollection or Config makes sense? But I am not so sure.

Reply:
Maybe something like `config.attach(User.create_root)` ?

Reply:
well, that already imply we need to check existence of root and do some actions. Do we ever use `#root` as nil? maybe UsersCollection.root can add root if it is not there yet?

Reply:
I would keep the collections agnostic to the business logic. I would delegate the responsibility of having a root or not to the user of the class (as it is now). The only thing I would like to have is a root template for the cases where we decide to add a root to the collection. Anyway, I think this detail can be addressed later, I don't want to block the PR only for that ;).